biggraphite/drivers/elasticsearch.py

```diff
@@ -577,6 +584,49 @@ def _search_metrics_from_components(self, glob, components, search=None):
                 search = search.filter(filter_type, **{"p%d" % i: value})
         return False, search
 
+    def _search_directory_from_components(self, glob, components, search=None):
+        """Assembles a query to search directory names.
+
+        Raises:
+          InvalidArgumentError: If the components include a globstar
+        """
+        # TODO (r.bizos) add unittest with directory index
+        glob_depth = _get_depth_from_components(components)
+        search = self._create_search_query() if search is None else search
```

Comment:
Could be simplified to 
`if search is None: search = self._create_search_query()`